lib/measures/tbd/resources/psi.rb

```diff
@@ -10,15 +11,43 @@
   require_relative "version.rb"
 end
 
+# sources for the following defaults KHI & PSI values/sets:
+
+# BETB = BC Hydro's Building Envelope Thermal Bridging Guide v1.4
+# www.bchydro.com/content/dam/BCHydro/customer-portal/documents/power-smart/
+# business/programs/BETB-Building-Envelope-Thermal-Bridging-Guide-v1-4.pdf
+
+# NECB-QC : Québec's energy code for new comemrcial buildings
+# www2.publicationsduquebec.gouv.qc.ca/dynamicSearch/telecharge.php?type=1&file=72541.pdf
+
+class KHI
+  # @return [Hash] KHI
+  attr_reader :point
+
+  def initialize
+    @point = {}
+
+    # The following are defaults (* stated). Users may edit these defaults,
+    # add new KHI pairs, or even read-in other KHI pairs on file.
+    # Units are in W/K.
+    @point[ "poor (BC Hydro)" ]         = 0.900.freeze # detail 5.7.2 BETB
+    @point[ "regular (BC Hydro)" ]      = 0.500.freeze # detail 5.7.4 BETB
+    @point[ "efficient (BC Hydro)" ]    = 0.150.freeze # detail 5.7.3 BETB
+    @point[ "code (Quebec)" ]           = 0.500.freeze # art. 3.3.1.3. NECB-QC
+    @point[ "(non thermal bridging)" ]  = 0.000.freeze
+  end
```

Comment:
If you are trying to make sure that people can't modify the hash I think you want `@point.freeze` after you set the defaults.  `0.900.freeze` just means you can't change that literal value which you couldn't anyway.

Reply:
Good call. I was doing it right for the PSI sets, but not here. Will correct.